src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -1069,8 +1069,143 @@ theorem rotateLeft_eq_rotateLeftAux_of_lt {x : BitVec w} {r : Nat} (hr : r < w)
     x.rotateLeft r = x.rotateLeftAux r := by
   simp only [rotateLeft, Nat.mod_eq_of_lt hr]
 
+
+/-- Accessing bits in `x.rotateLeft r` the range `[0, r)` is equal to
+    accessing bits `x` in the range `[w - r, w)`,
+    given by `i : [0, r) ↦ w - r + i : [w - r, w)`.
```

Comment:
```suggestion
/--
Accessing bits in `x.rotateLeft r` the range `[0, r)` is equal to
accessing bits `x` in the range `[w - r, w)`,
given by `i : [0, r) ↦ w - r + i : [w - r, w)`.
```
and below. Life is too short to maintain indenting. :-)

Reply:
To be honest, I found this explanation (particularly that it uses lots of non-Lean notation!) less helpful than the theorem statement. I like examples in doc-strings, but this one needs a rewrite!

Reply:
Fixed!